issue_processing/issue_processing.py

```diff
@@ -242,3 +461,41 @@ def print_to_disk(issues, results_folder):
 
     # write to output file
     csv_writer.write_to_csv(output_file, lines)
+
+
+def print_to_disk_new(issues, results_folder):
+    """Print issues to file 'issues_new.list' in result folder.
+    This file has a consistent format to the 'bugs-jira.list' file.
+    When the network library is updated, this is the format which shall be used.
```

Comment:
Please add a `TODO` in front of this line, so we can easily re-find this comment.